website_event_compassion/models/event_registration.py

```diff
@@ -629,6 +638,7 @@ def prepare_group_visit_payment(self):
             'name': product.name,
             'product_id': product.id,
             'account_analytic_id': event.analytic_id.id,
+            'payment_mode_id': mode_pay_bvr.id,
```

Comment:
This is invoice line, it doesn't need payment_mode

Reply:
You should add the payment_mode just a bit below, at line 639 (when creating the invoice)